playground/TestShop/AppHost/Program.cs

```diff
@@ -17,9 +17,11 @@
                            .WithReference(basketCache)
                            .WithReference(messaging);
 
+var isHttps = string.Equals(builder.Configuration["DOTNET_LAUNCH_PROFILE"], "https", StringComparison.OrdinalIgnoreCase);
+
 builder.AddProject<Projects.MyFrontend>("frontend")
        .WithReference(basketService)
-       .WithReference(catalogService.GetEndpoint("http"));
+       .WithReference(catalogService.GetEndpoint(isHttps ? "https" : "http"));
```

Comment:
I assume we want to keep the explicit `.GetEndpoint` call in here so I updated it to ensure it works whether the project is run the `http` or `https` profile.

Reply:
I don't see a need for this explicit call, but I'm not sure why it was here to begin with.

Reply:
I assumed it was here as a "scream test" kind of thing? Just to make sure the API was used in TestShop. I'm happy to remove it.

Reply:
It's cleaner if we delete it. It looks like a paper cut, but it's really just unnecessary.

Reply:
OK I'll delete it.